test/e2e/serial/tests/workload_unschedulable.go

```diff
@@ -220,4 +220,133 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload unsched
 			}
 		})
 	})
+
+	Context("with at least one node", func() {
+		It("[test_id:47616][tier2][tmscope:pod] pod with two containers each on one numa zone can NOT be scheduled", func() {
+
+			// Filter by number of numa zones
+			const requiredNUMAZones = 2
```

Comment:
This still feels a bit awkward in go. Maybe just
```
By(fmt.Sprintf("filtering available nodes with at least %d NUMA zones", 2))
nrtCandidates := e2enrt.FilterZoneCountEqual(nrtList.Items, 2)
```
(same for the other `const` right below)

Reply:
what about using 
`requiredNUMAZones := 2` 
to avoid repeating "magic" numbers?

Reply:
yes, we can do this